site/docs/embed.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: docs
+menu: docs
+title: Embed
+permalink: /docs/embed.html
+---
+
+This Document assumes that you have a Vega-Lite spec as a JavaScript object in a variable called `vlSpec`. You can find a simple example in our [tutorial]({{site.baseurl}}/tutorials/getting_started.html#embed).
+
+## Load the Required Libraries
+
+To use Vega-Lite on your own web page, load the required libraries (D3, Vega, and Vega-Lite). You can optionally load vega-embed to enable the convenient `vg.embed` method.
+
+```html
+<script src="//d3js.org/d3.v3.min.js" charset="utf-8"></script>
+<script src="//vega.github.io/vega/vega.js" charset="utf-8"></script>
+<script src="//vega.github.io/vega-lite/vega-lite.js" charset="utf-8"></script>
+<script src="//vega.github.io/vega-editor/vendor/vega-embed.js" charset="utf-8"></script>
+```
+
+You can also download Vega-Lite: [vega-lite.min.js]({{site.baseurl}}/vega-lite.min.js).
```

Comment:
Since we might release docs more often that actual release, should we link to the latest release file instead?